ironic_bug_dashboard/__init__.py

```diff
@@ -14,37 +16,28 @@
 template_path = os.path.dirname(os.path.realpath(__file__))
 aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(template_path))
 
+parser = argparse.ArgumentParser(description='Load configuration from commandline.')
+parser.add_argument('project_name', nargs='?', type=str, help='Name of the project.')
+args, unknown_args = parser.parse_known_args()
+
+config = simple_lp.load_config(args.project_name)
+if config is None or len(config) <= 0:
+    LOG.error('Configuration file cannot be empty.')
+    sys.exit()
+
+LOG.info("*" * 80)
+LOG.info("Configuration options gathered from:")
+LOG.info("command line args: %s", args)
+LOG.info("config file: %s", args.project_name)
```

Comment:
This log statement can perhaps be moved to the simple_lp.load_config function with the actual path to the config file being used. 
To keep the sequence of the output log statements the same, the log statements on lines 28-30 can be moved before the call to the load_config function.

Reply:
So, if I understand correctly, you're suggesting I move lines 28-30 before the call to the load_config function, and 31-32 into the simple_lp.load_config function definition where the actual path to the config file can be logged?

Reply:
Updated